### ISSUE TITLE
Update MAINTAINERS.md

### ISSUE THREAD

**Issue by beorn7** [prometheus/common#147]
Current MAINTAINERS.md lists @fabxc as the maintainer, but with his CoreOS mail address (which presumably wouldn't work anymore as he doesn't work for CoreOS anymore).

There has been very little commit activity an this repo recently, but there are a bunch of open PRs, new and old, and a lot of meaningful issues, again new and old. Most of recent activities on issues and PRs is (unsurprisingly) by @brian-brazil . The current maintainer doesn't seem to have shown any recent activity here.

In short: This repo appears abandoned. However, it is used by many others across the project, thus having this properly maintained is crucial. Many of the changes here, and in particular some of the long-standing open issues, touch quite basic decisions that have impact on the project as a whole and need consistency across the project (including the meta issue of what should go into `common` at all – the different opinions about this have been colliding here and there over the last years, up to the current day… #97 is just the tip of the iceberg). I would love to see somebody who is not only actively maintaining this but also has the ability to mediate all those basic decisions into a productive consensus.

**Comment by juliusv** [comment]
@fabxc Could you provide an updated email address? I assume you would be fine with a maintainership change if someone volunteered (which doesn't seem like the case right now)?

**Comment by fabxc** [comment]
I advocated for phasing out this repo as much as possible, others argued for the opposite and more functionality has been added since. I understand arguments for either and don't feel too strongly but can't really be a maintainer for things I didn't add and wouldn't agree to add.

I think we discussed this to the same extent some time ago on IRC. I suggest @brian-brazil takes maintainership here since he was the main advocate for centralising config structs etc. here.
